managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java

```diff
@@ -341,8 +341,8 @@ public void recycle() {
     @Override
     public String toString() {
         return "OpAddEntry{" +
-                "mlName" + ml.getName() +
-                "ledgerId=" + ledger.getId() +
+                "mlName=" + ml != null ? ml.getName() : "null" +
+                "ledgerId=" + ledger != null ? String.valueOf(ledger.getId()) : "null" +
```

Comment:
It's better to add a comma
```suggestion
                ", ledgerId=" + ledger != null ? String.valueOf(ledger.getId()) : "null" +
```